lib/binary_to_decimal.rb

```diff
@@ -5,5 +5,11 @@
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
 def binary_to_decimal(binary_array)
```

Comment:
This method works, however can you get it to work without the Enumerable method `reverse`.